src/load_order.ts

```diff
@@ -665,14 +826,59 @@ const deployAndSerializeNewLoadOrder: VortexWrappedSerializeFunc = (
   const loID = Date.now();
 
   const v2077LoadOrder = makeV2077LoadOrderFrom(vortexLoadOrder, ownerVortexProfileId, loID);
+  const modsDotJsonLoadOrder = makeModsDotJsonLoadOrderFrom(vortexLoadOrder);
 
   vortexApi.log(`info`, `${me}: New load order ${loID} ready to be deployed and serialized!`);
   vortexApi.log(`debug`, `${me}: Load order ${loID}:`, S(v2077LoadOrder));
 
+  let maybeSuccessfullyReWroteModsDotJsonToDisk: TaskEither<{ stdout: string; stderr: string; }, void>;
   // We want to wait until there's been a deployment - either the automatic one
   // from an enable or something like that, or a manually triggered one.
   vortexApi.events.once(`did-deploy`, () => {
-    startREDmodDeployInTheBackgroundWithNotifications(vortexApi, gameDirPath, loID, v2077LoadOrder, vortexLoadOrder);
+    maybeSuccessfullyReWroteModsDotJsonToDisk =
+    pipe(
+      tryCatchTE(
+        () =>
+          startREDmodDeployInTheBackgroundWithNotifications(vortexApi, gameDirPath, loID, v2077LoadOrder, vortexLoadOrder),
+        (err) => {
+          vortexApi.log(`error`, `${me}: REDmod deployment ${loID} failed!`, S(err));
+          showInfoNotification(vortexApi, InfoNotification.REDmodDeploymentFailed);
+          throw err;
+        },
+      ),
+      mapTE(({ stdout, stderr: _stderr }) => {
```

Comment:
Tiny bit of additional semantic context if you use [`chain*`](https://gcanti.github.io/fp-ts/modules/TaskEither.ts.html#chain-1)